compiler/daml-extension/src/extension.ts

```diff
@@ -86,6 +94,17 @@ export async function activate(context: vscode.ExtensionContext) {
     context.subscriptions.push(d1, d2, d3, d4, d5);
 }
 
+// Compare the extension version with the one stored in the global state.
+// This will be used to show release notes when the version has changed.
+async function checkVersion(context: ExtensionContext) {
+    let packageFile = path.join(context.extensionPath, 'package.json');
+    let data = fs.readFileSync(packageFile, "utf8");
```

Comment:
Probably better to use https://stackoverflow.com/a/46867579/3877993 to turn `readFile` into an `async` (depending on which node version you use, you don’t even need that https://stackoverflow.com/a/56821924/3877993). `readFileSync` looks up the whole js runtime which is probably fine since VSCode sandboxes extensions but given that we can easily avoid it here, let’s do that.